cards.js

```diff
@@ -0,0 +1,80 @@
+import dinosaurs from "./dinosaurs.json" assert { type: "json" };
+console.log(dinosaurs)
+
+const fetchData = async () =>{
+    try{
+        const res = await fetch("https://chinguapi.onrender.com/dinosaurs");
+        const data = await res.json();
+        console.log(data)
+        return data
+    }catch(err){
+        console.log("Fetch Error",err);
+    }
+}
+
+const dinoContainer = document.getElementById('all-dinos-container');
+
+const getSingleDino = async (dinoId) => {
+    try {
+        const response = await fetch(`${dinosaurId}/${dinoId}`)
+        console.log(response)
+        const dino = await response.json();
+        console.log(dino)
+        return dino
+    } catch {
+        // console.error(`trouble getting dinosaur #${dinoId}!`);
+    }
+};
```

Comment:
Since you aren't using this code, please delete.

Reply:
OOPS!!  I accidentally included line 15, which is necessary code.  So this is an example of why you should remove unused code and test again instead of another team member.